tests/test_routes.py

```diff
@@ -141,6 +141,337 @@ def test_cves_returns_422_for_non_existing_package_status(self):
         assert response.status_code == 422
         assert "Cannot find a status" in response.json["errors"]
 
+    def test_cve_updated_at_column(self):
+        """
+        Tests that updated_at is created automatically when a new CVE is added
+        and that it is changed on update.
+        """
+
+        cve_payload = payloads.cve1.copy()
+
+        add_cve_response = self.client.put(
+            "/security/cves.json",
+            json=[cve_payload],
+        )
+
+        assert add_cve_response.status_code == 200
+
+        cve = self.client.get(f"/security/cves/{cve_payload['id']}.json").json
+
+        # Check that value is None when CVE is first added
+        assert cve["updated_at"] is None
```

Comment:
I'm mildly surprised by this. For some reason I would have expected `updated_at` to simply carry the created date initially. I'm not quite sure whether this opinion has any basis...

Reply:
Originally that's how I had it, but then when it came time to order by update date having it carry the creation date created incorrect sorting. Lmk if you think there's a better way to handle that though!

Reply:
So say we have a CVE that was added June 10th at 3pm, and another that was updated June 10th at 2pm. If `updated_at` carries the creation date on default then when you try to sort by update date the returned list suggests that that the first CVE was updated June 10th at 3pm and that it was updated more recently than the second CVE, both of which are not correct

Reply:
Do you think that's not correct? I totally agree we shouldn't populate "updated_at" with the created date if we don't believe that to be semantically true. So the disagreement here is a semantic one - in your above scenario, which would we expect to be returned. Personally, I consider "creation" to be within the broad definition of "update" and I would expect the 3pm CVE to be returned first.

Reply:
> We can always compare publication date and last updated to check if info has been updated after being first published, if there's the need for it (e.g. hiding the "Last updated" date on the website when there's no need for it).

Yes I think this is a better way to handle this particular situation. If we agree on the sorting point, then I think it's better to go for a database schema that avoids `NULL`s if possible.